examples/k8s.yaml

```diff
@@ -55,8 +55,9 @@ provision:
     export DEBIAN_FRONTEND=noninteractive
     apt-get update
     apt-get install -y apt-transport-https ca-certificates curl
-    curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://dl.k8s.io/apt/doc/apt-key.gpg
-    echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+    VERSION=$(curl -L -s https://dl.k8s.io/release/stable.txt | sed -e 's/v//' | cut -d'.' -f1-2)
+    echo "deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v${VERSION}/deb/ /" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+    curl -fsSL https://pkgs.k8s.io/core:/stable:/v${VERSION}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
```

Comment:
Not new in this PR, but we have to check the fingerprint or maybe just sha256, otherwise signing isn't really meaningful

Reply:
Actually I just copied the upstream instructions literally, so it could be worth a PR to kubernetes if that is so ?

Reply:
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl

Reply:
The signing key has been known to change, when you least expect it.

Reply:
This is the information that I have about it: https://build.opensuse.org/projects/isv:kubernetes/signing_keys

Reply:
- https://github.com/kubernetes/website/issues/42718

Reply:
Here was the previous commit, when moving the key url to the download server: https://github.com/kubernetes/website/commit/70266d5f79db1fc8f3db7bc04571bf0725789b05